### ISSUE TITLE
onnx.optimizer has no attribute 'get_available_passes'

### ISSUE THREAD

**Issue by elboyran** [onnx/onnx#1299]
Installing onnx from binaries, e.g. using
`conda install -c conda-forge onnx.`
Running the [optimize_onnx.ipynb](https://github.com/onnx/onnx/blob/master/onnx/examples/optimize_onnx.ipynb) notebook from the Python API Overview, second cell gives an error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-531bae708a9a> in <module>()
      1 # A full list of supported optimization passes can be found using get_available_passes()
----> 2 all_passes = optimizer.get_available_passes()
      3 print("Available optimization passes:")
      4 for p in all_passes:
      5     print(p)

AttributeError: module 'onnx.optimizer' has no attribute 'get_available_passes'
```

The same is observed from a command line:

```
>>> from onnx import optimizer
>>> all_passes = optimizer.get_available_passes()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'onnx.optimizer' has no attribute 'get_available_passes'
```

**Comment by houseroad** [comment]
Because `onnx.optimizer.get_available_passes` is just added. If you would like to run optimize_onnx.ipynb, please install from the source code.

**Comment by elboyran** [comment]
Installing from source did not work. I've made a new issue about that: #1302 

Installing with pip did work (as I commented in issue #1302 ), and solved the #1298 issue. It seems like the optimizer function seems is still not added in the pip installation as I keep getting the same error.

**Comment by elboyran** [comment]
Installing from source did not work. I've made a new issue about that: #1302 

Installing with pip did work (as I commented in issue #1302 ), and solved the #1298 issue. It seems like the optimizer function seems is still not added in the pip installation as I keep getting the same error.
